addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -93,6 +93,11 @@ def test_message_parse_xhtml(self):
         # Test that the parsing of XHTML mails does not fail
         self.env['mail.thread'].message_parse(self.from_string(test_mail_data.MAIL_XHTML))
 
+    def test_message_parse_wrong_content_charset(self):
```

Comment:
Technically this should be defined before test_message_parse_xhtml (alphabetical ordering so that it matches executio order :) )